packages/ckeditor5-image/theme/image.css

```diff
@@ -34,6 +34,10 @@
 			display: block;
 			max-width: 100%;
 		}
+
+		&.ck-widget_selected {
```

Comment:
It's not a content style but an editing style. Integrators don't need it in their [content (front-end) styles](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/content-styles.html
). It should go under `.ck.ck-editor__editable .image-inline` similar to placeholder styles blow :point\_down:.

Reply:
:sweat_smile: